scripts/start/readCheckScenarioConfig.R

```diff
@@ -27,18 +27,27 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
             paste0(rownames(scenConf)[regionname], collapse = ", "),
             " – Titles with three capital letters or 'glob' may be confused with region names by magclass. Stopping now.")
   }
-  containsdot <- grep("\\.", rownames(scenConf))
+  containsdot <- grepl("\\.", rownames(scenConf))
   if (any(containsdot)) {
     warning("These titles contain a dot: ",
             paste0(rownames(scenConf)[containsdot], collapse = ", "),
             " – GAMS would not tolerate this, and quit working at a point where you least expect it. Stopping now.")
   }
+  whitespaceerrors <- 0
+  for (path_gdx in intersect(names(path_gdx_list), names(scenConf))) {
+    haswhitespace <- grepl(" ", scenConf[[path_gdx]])
```

Comment:
```suggestion
    haswhitespace <- grepl("\\s", scenConf[[path_gdx]])
```
Will also match on [protected spaces](https://en.wikipedia.org/wiki/Non-breaking_space) and other stuff people can come up with to break remind.